parsers/IN.py

```diff
@@ -247,31 +254,23 @@ def fetch_npp_production(
                 "TODAY'S\nACTUAL\n": "value",
             }
         )
-        df_npp = df_npp[["power_station", "production_mode", "value"]]
-        df_npp = df_npp.iloc[1:].copy()
-        df_npp["production_mode"] = df_npp["production_mode"].ffill()
-
         df_npp["region"] = (
             df_npp["power_station"]
             .apply(lambda x: NPP_REGION_MAPPING[x] if x in NPP_REGION_MAPPING else None)
             .ffill()
         )
-        df_zone = df_npp.loc[df_npp["region"] == zone_key].copy()
-        df_zone = df_zone.loc[~df_zone.power_station.isna()]
-        df_zone = df_zone[df_zone.power_station.str.contains("TYPE:")]
-        df_zone = df_zone[["production_mode", "value"]]
-
-        production_in_zone = df_zone.groupby(["production_mode"]).sum()
-        production = {}
-        for mode in production_in_zone.index:
-            production[NPP_MODE_MAPPING[mode]] = round(
-                production_in_zone.iloc[
-                    production_in_zone.index.get_indexer_for([mode])[0]
-                ].get("value")
-                / CONVERSION_GWH_MW,
-                3,
-            )
-        return production
+        df_npp = df_npp[["region", "production_mode", "value"]]
+
+        df_npp_filtered = df_npp.loc[~df_npp["production_mode"].isna()].copy()
```

Comment:
I am not sure to understand why this fixes the issue we've seen before

Reply:
![image](https://user-images.githubusercontent.com/107848894/235661613-6f170314-e4c7-41e6-aa5e-aad13b43eb2a.png)
When there is value in this col it corresponds to the total for the given mode for all the PPs in the sector in the state. This way we don't have to filter on "TYPE:" which may or may not be in the column text